map/framework.cpp

```diff
@@ -1897,10 +1906,21 @@ df::SelectionShape::ESelectedObject Framework::OnTapEventImpl(df::TapInfo const
     return df::SelectionShape::OBJECT_USER_MARK;
   }
 
+  FeatureID featureTapped = tapInfo.m_featureTapped;
+
+  if (!featureTapped.IsValid() && df::GetDrawTileScale(m_currentModelView) >= m_minBuildingsTapZoom)
```

Comment:
Всмысле, у него нет возможности обратиться к индексу?

> On 22.3.2016, at 17.01, Daria Volvenkova <notifications@github.com> wrote:
> 
> In map/framework.cpp:
> 
> > @@ -1897,10 +1906,21 @@ df::SelectionShape::ESelectedObject Framework::OnTapEventImpl(df::TapInfo const
> >      return df::SelectionShape::OBJECT_USER_MARK;
> >    }
> >  
> > +  FeatureID featureTapped = tapInfo.m_featureTapped;
> > +
> > +  if (!featureTapped.IsValid() && df::GetDrawTileScale(m_currentModelView) >= m_minBuildingsTapZoom)
> Сейчас дрейп оперирует только обезличенной геометрией и оверлеями, другой информации о зданиях у него нет.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>